src/components/shared/Icon.css

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.nodeIcon {
+  min-width: 14px;
+  height: 100%;
+  min-height: 14px;
+  background: center / contain no-repeat;
+}
```

Comment:
It looks like these changes belong to your previous PR. It's landed already, so we shouldn't see them here anymore. That would be better if you can open new branch from main instead of your previous branch. So that way we can only see the relevant commits. That can cause some merge conflicts after the previous ones are merged but we need to fix them anyway :)

Reply:
I did all necessary fixes and resolved conflicts. I think now it should be ready to go?
Because of the merge that I did for be able to solve the conflict, all other commits made by other contributors are seen in this pull request. I hope It wouldn't affect this request, otherwise I can open a new request. @canaltinova